Cython/Includes/cpython/dict.pxd

```diff
@@ -3,11 +3,91 @@ from .pyport cimport uint64_t
 
 cdef extern from *:
     # On Python 2, PyDict_GetItemWithError is called _PyDict_GetItemWithError
+    # Also backport PyDict_GetItemStringRef and PyDict_SetDefaultRef
     """
     #if PY_MAJOR_VERSION <= 2
     #define PyDict_GetItemWithError _PyDict_GetItemWithError
     #endif
+
+    #if PY_VERSION_HEX < 0x030D0000
```

Comment:
We generally spell this with a lower-case `d` because `D` is easily confused with `0` in some fonts.
```suggestion
    #if PY_VERSION_HEX < 0x030d0000
```